src/account/account.cairo

```diff
@@ -69,7 +70,13 @@ mod AccountComponent {
             // Check tx version
             let tx_info = get_tx_info().unbox();
             let tx_version: u256 = tx_info.version.into();
-            assert(MIN_TRANSACTION_VERSION <= tx_version, Errors::INVALID_TX_VERSION);
+            if (tx_version >= QUERY_OFFSET) {
```

Comment:
```suggestion
            if (tx_version >= QUERY_OFFSET) {
```
do you think a small single-line comment explaining this could help?

Reply:
Yes, indeed